api/src/main/java/org/pmiops/workbench/db/model/DbCdrVersion.java

```diff
@@ -452,35 +460,50 @@ public String getWgsLongReadsHailGRCh38() {
     return wgsLongReadsHailGRCh38;
   }
 
-  public void setWgsLongReadsHailGRCh38(String wgsLongReadsHailGRCh38) {
+  public DbCdrVersion setWgsLongReadsHailGRCh38(String wgsLongReadsHailGRCh38) {
     this.wgsLongReadsHailGRCh38 = wgsLongReadsHailGRCh38;
+    return this;
   }
 
   @Column(name = "wgs_longread_hail_t2t")
   public String getWgsLongReadsHailT2T() {
     return wgsLongReadsHailT2T;
   }
 
-  public void setWgsLongReadsHailT2T(String wgsLongReadsHailT2T) {
+  public DbCdrVersion setWgsLongReadsHailT2T(String wgsLongReadsHailT2T) {
     this.wgsLongReadsHailT2T = wgsLongReadsHailT2T;
+    return this;
   }
 
   @Column(name = "wgs_longread_joint_vcf_grch38")
   public String getWgsLongReadsJointVcfGRCh38() {
     return wgsLongReadsJointVcfGRCh38;
   }
 
-  public void setWgsLongReadsJointVcfGRCh38(String wgsLongReadsJointVcfGRCh38) {
+  public DbCdrVersion setWgsLongReadsJointVcfGRCh38(String wgsLongReadsJointVcfGRCh38) {
     this.wgsLongReadsJointVcfGRCh38 = wgsLongReadsJointVcfGRCh38;
+    return this;
   }
 
   @Column(name = "wgs_longread_joint_vcf_t2t")
   public String getWgsLongReadsJointVcfT2T() {
     return wgsLongReadsJointVcfT2T;
   }
 
-  public void setWgsLongReadsJointVcfT2T(String wgsLongReadsJointVcfT2T) {
+  public DbCdrVersion setWgsLongReadsJointVcfT2T(String wgsLongReadsJointVcfT2T) {
     this.wgsLongReadsJointVcfT2T = wgsLongReadsJointVcfT2T;
+    return this;
+  }
+
+  @Column(name = "needs_v8_genomic_extraction_workflow")
+  public Boolean getNeedsV8GenomicExtractionWorkflow() {
```

Comment:
why we need this stored in DB? Can this is done by checking CDR version in our API code? 
`dbWorkspace.getCdrVersion === 8`

Reply:
There is no existing field which corresponds to what we consider a CDR release.  See discussion at https://pmi-engteam.slack.com/archives/C0189PPPKFF/p1730391488507559

Reply:
Trying to understand the CdrRelease vs actual cdrVersion of a workspace.
Which workflow should be decided by the version that a workspace is using, so that is versionId ?

Reply:
saw your reply in slack, that's helpful. 1st time to know CDR release vs data release. 
Then if we chose this option, how that DB field will be updated? A backfill or something we can figure out when user create WS?